openedx_tagging/core/tagging/rest_api/v1/serializers.py

```diff
@@ -216,3 +217,31 @@ class TaxonomyTagDeleteBodySerializer(serializers.Serializer):  # pylint: disabl
         child=serializers.CharField(), required=True
     )
     with_subtags = serializers.BooleanField(required=False)
+
+
+class TaxonomyImportBodySerializer(serializers.Serializer):  # pylint: disable=abstract-method
+    """
+    Serializer of the body for the Taxonomy Import action
```

Comment:
```suggestion
    Serializer of the body for the Taxonomy Import request
```
I think it's important to distinguish between request and response serializers, when it's only one or the other.